packages/mermaid/src/docs/ecosystem/integrations.md

```diff
@@ -199,3 +199,14 @@ They also serve as proof of concept, for the variety of things that can be built
 - [NiceGUI: Let any browser be the frontend of your Python code](https://nicegui.io)
   - [ui.mermaid(...)](https://nicegui.io/reference#mermaid_diagrams)
   - [ui.markdown(..., extras=['mermaid'])](https://nicegui.io/reference#markdown_element)
+
+## Recommendations
+
+### File Extension
+
+Applications that support editing and saving mermaid files are encouraged to use the `.mmd` file extension.
```

Comment:
```suggestion
Applications that support editing and saving mermaid files are encouraged to use `.mermaid` or `.mmd` file extension.
```

Reply:
Added an `s`, but think the intent is there. Also feel free to make any changes directly! Just figured something concrete would get the ball rolling!